webhook.rst

```diff
@@ -0,0 +1,132 @@
+Webhook
+=======
+
+.. versionadded:: 6.3
+
+    The Webhook component was introduced in Symfony 6.3
+
+The Webhook component is used to respond to remote webhooks to trigger actions
+in your application. This document focuses on using webhooks to listen to remote
+events in other Symfony components.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/webhook
+
+Usage in combination with the Mailer component
+----------------------------------------------
+
+When using a third-party mailer, you can use the Webhook component to receive
+webhook calls from the third-party mailer.
+
+In this example Mailgun is used with ``'mailer_mailgun'`` as webhook type.
+Any type name can be used as long as it's unique. Make sure to use it in the
+routing configuration, the webhook URL and the RemoteEvent consumer.
+
+Install the third party mailer as described in the documentation of the
+:doc:`Mailer component </mailer>`.
```

Comment:
```suggestion
:doc:`Mailer component </mailer_using-a-3rd-party-transport>`.
```

@OskarStark would this notation work to link directly to that part of the documentation?